doc/3/controllers/realtime/subscribe/index.md

```diff
@@ -52,10 +52,3 @@ _Subscription to document notifications with scope option_
 
 <<< ./snippets/document-notifications-leave-scope.java
 
-_Subscription to message notifications_
```

Comment:
why is this removed?

Reply:
Those lines are a mistake from the realtime controller PR, those snippets do not exists